pkgs/applications/editors/jetbrains/versions.json

```diff
@@ -3,56 +3,56 @@
-      "sha256": "55b28f3b79eda126fe778e2945804d50b1145503737f1b5e25ab6ae2d2a0e3ae",
-      "url": "https://download.jetbrains.com/datagrip/datagrip-2022.2.5.tar.gz",
+      "version": "2022.3.2",
+      "sha256": "e542111e490fbbc80d3aebcbbc343b29e17bf6766d7b708675618d8e49b6ee83",
+      "url": "https://download.jetbrains.com/datagrip/datagrip-2022.3.2.tar.gz",
       "version-major-minor": "2022.1.1",
-      "build_number": "222.4345.5"
+      "build_number": "223.8214.62"
     },
     "gateway": {
       "update-channel": "Gateway EAP",
       "url-template": "https://download.jetbrains.com/idea/gateway/JetBrainsGateway-{version}.tar.gz",
-      "version": "2022.3 EAP",
-      "sha256": "4868baed9350065c1db760f07a09badd1473132af640cc19330e20c8a0940d7d",
-      "url": "https://download.jetbrains.com/idea/gateway/JetBrainsGateway-223.6646.21.tar.gz",
+      "version": "2022.3 Beta",
+      "sha256": "d2c3a0e2d521ce5ac542b86cc9bf1e35c484708ca15f546df59bd42c64290eb7",
+      "url": "https://download.jetbrains.com/idea/gateway/JetBrainsGateway-223.7571.4.tar.gz",
       "version-major-minor": "2022.3",
-      "build_number": "223.6646.21"
+      "build_number": "223.7571.4"
     },
     "goland": {
       "update-channel": "GoLand RELEASE",
       "url-template": "https://download.jetbrains.com/go/goland-{version}.tar.gz",
-      "version": "2022.2.4",
-      "sha256": "e39aaae39e6021e87cece7622c51860d23e2a5b5ac2683fb67d369ec7d609084",
-      "url": "https://download.jetbrains.com/go/goland-2022.2.4.tar.gz",
+      "version": "2022.3.1",
+      "sha256": "566eada40511cd06727d69047e8a6a1e75b06ebade93d1ea78262fc2715c8a38",
+      "url": "https://download.jetbrains.com/go/goland-2022.3.1.tar.gz",
       "version-major-minor": "2022.2",
-      "build_number": "222.4345.24"
+      "build_number": "223.8214.59"
     },
     "idea-community": {
       "update-channel": "IntelliJ IDEA RELEASE",
       "url-template": "https://download.jetbrains.com/idea/ideaIC-{version}.tar.gz",
-      "version": "2022.2.3",
-      "sha256": "4ba5faafad48d58db5099fae080ae2238086d3d9803080082de8efe35d8bf4ed",
-      "url": "https://download.jetbrains.com/idea/ideaIC-2022.2.3.tar.gz",
+      "version": "2022.3.1",
+      "sha256": "4c3514642ce6c86e5343cc29b01c06ddc9c55f134bcb6650de5d7d36205799e8",
+      "url": "https://download.jetbrains.com/idea/ideaIC-2022.3.1.tar.gz",
       "version-major-minor": "2022.1",
-      "build_number": "222.4345.14"
+      "build_number": "223.8214.52"
     },
     "idea-ultimate": {
       "update-channel": "IntelliJ IDEA RELEASE",
       "url-template": "https://download.jetbrains.com/idea/ideaIU-{version}-no-jbr.tar.gz",
-      "version": "2022.2.3",
+      "version": "2022.3.1",
       "sha256": "7454d7e0b8f4e3d8d805dde645d28b842101bd77aea8b29125880c592e6b8c85",
-      "url": "https://download.jetbrains.com/idea/ideaIU-2022.2.3-no-jbr.tar.gz",
+      "url": "https://download.jetbrains.com/idea/ideaIU-2022.3.1-no-jbr.tar.gz",
```

Comment:
404, also can't find a related link in https://www.jetbrains.com/idea/download/other.html
I found https://youtrack.jetbrains.com/issue/IDEA-309170:
> That's correct; starting from 2022.3, we're no longer providing no-JBR distributions and corresponding patches. 
> I'll check what we can do to make the user experience better.

Reply:
Jetbrains seems to no more support with no-JBR as they are patching OpenJDK feature on their side.  
Replacing this link seems to be the best to do.